examples/options/include/traccc/options/options.hpp

```diff
@@ -21,7 +21,7 @@
 namespace traccc {
 
 namespace {
-constexpr char s_separator = ':';
+const std::string s_separator = ":";
```

Comment:
Since this thing is in the header file, let's make it into a `static const` variable. I wonder if `static constexpr` would even work... :thinking:

The private namespace does a very similar sort of thing, but I'm just partial to using `static` on such constants.

Reply:
It would need to be a `std::string_view`, then it can be `static constexpr`.

Reply:
I changed to `static constexpr std::string_view` and also added a simple unit test for option parsing